mavros_msgs/msg/ExtendedState.msg

```diff
@@ -0,0 +1,15 @@
+# Extended autopilot state
```

Comment:
I think to add link to mavlink doc. e.g.

https://pixhawk.ethz.ch/mavlink/#EXTENDED_SYS_STATE